tests/managed-service/test_acceptance_managed_service.py

```diff
@@ -0,0 +1,153 @@
+import logging
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import ManageTest, managed_service_required
+from ocs_ci.ocs.resources import pvc
+from ocs_ci.utility.utils import TimeoutSampler
+from ocs_ci.helpers import helpers
+
+logger = logging.getLogger(__name__)
+
+
+class TestAcceptanceMS(ManageTest):
+    """
+    Acceptance test Managed Service
+
+    """
+
+    @managed_service_required
+    def test_acceptance_managed_service(
```

Comment:
isn't this test case applicable for non MS clusters too?

Reply:
This test applicable for non-MS clusters too